src/Uno.UI/UI/Xaml/Controls/AppBar/AppBarButtonTemplateSettings.cs

```diff
@@ -0,0 +1,15 @@
+
+namespace Windows.UI.Xaml.Controls.Primitives
+{
+	public  partial class AppBarButtonTemplateSettings : DependencyObject
+	{
+		public double KeyboardAcceleratorTextMinWidth
+		{
+			get
+			{
+				global::Windows.Foundation.Metadata.ApiInformation.TryRaiseNotImplemented("Windows.UI.Xaml.Controls.Primitives.AppBarButtonTemplateSettings", "KeyboardAcceleratorTextMinWidth");
```

Comment:
should this `TryRaiseNotImplemented` be removed?

Reply:
Those two really are not implemented.

Reply:
should be marked `[NotImplemented]`, then